libs/application/templates/aosh/work-accident-notification/src/fields/TypeOfInjury/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { FieldBaseProps } from '@island.is/application/types'
+import { FC } from 'react'
+import { causeAndConsequences } from '../../lib/messages'
+import { WorkAccidentNotification } from '../../lib/dataSchema'
+import { CausesAndEffects } from '../Components/CausesAndEffects'
+
+interface TypeOfInjuryProps {
+  field: {
+    props: {
+      index: number
+    }
+  }
+}
+
+export const TypeOfInjury: FC<
+  React.PropsWithChildren<TypeOfInjuryProps & FieldBaseProps>
+> = (props) => {
+  const { application } = props
+  const answers = application.answers as WorkAccidentNotification
+  const idx = props.field?.props?.index
```

Comment:
```suggestion
interface TypeOfInjuryProps extends FieldBaseProps {
  field: {
    props: {
      index: number
    }
  }
}

export const TypeOfInjury: FC<
  React.PropsWithChildren<TypeOfInjuryProps>
> = ({application, field }) => {
  const answers = application.answers as WorkAccidentNotification
  const idx = field?.props?.index
```
I think this is a bit more readable but I'm also not a fan of the whole FC<... thing. But that is probably more of a me thing...

I would personally write this like this:
```
interface Props extends FieldBaseProps {
  field: {
    props: {
      index: number
    }
  }
}

export const TypeOfInjury = ({application, field }: Props) => {
  const answers = application.answers as WorkAccidentNotification
  const idx = field?.props?.index
```

Reply:
I don't mind either FC or not. I just kept the same style as the other AOSH applications. I don't destructure props in this case cause I spread props into the component below